fastlane/Fastfile

```diff
@@ -128,27 +128,45 @@ platform :ios do
   lane :lint_module do
     if is_ci?
       type = 'pmd'
-      path = './artifacts/lint-results.xml'
+      ocpath = './artifacts/lint-results.xml'
     else
       type = 'html'
-      path = './artifacts/lint-results.html'
+      ocpath = './artifacts/lint-results.html'
     end
 
+    swiftpath = './swiftlint.result.json'
```

Comment:
I don't think our CI will add this file to the job artifacts. 
It should be saved to `./artifacts` and the inapp CI job should be updated to archive the files in that folder - see the analytics job for an example.